### PR TITLE
Add as author BR

### PR DESCRIPTION
**Pull request by rbharath**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
The formatting looks good @rbharath ! I just wanted to check if the DeepChem Project has an address we could list.

**Comment by rbharath** [comment]
Great! 

No, it's a purely online effort so no physical address. I could add the website? https://deepchem.io/

**Comment by rando2** [comment]
@rbharath It sounds like a super cool initiative! A URL sounds good to me if it sounds good to you!

**Comment by rbharath** [comment]
Great! Just added on the URL

File: content/metadata.yaml [hunk 1]
```diff
@@ -210,3 +210,18 @@ authors:
...
+    name: Bharath Ramsundar
+    initials: BR
+    orcid: 0000-0001-8450-4262
+    twitter: rbhar90
+    email: bharath.ramsundar@gmail.com
+    contributions:
+      - Investigation
+      - Writing - Review & Editing
+    affiliations:
+      - The DeepChem Project (https://deepchem.io/)
```
Comment (rando2) [inline content/metadata.yaml:224]:
```suggestion
      - The DeepChem Project, https://deepchem.io/
```

**Pull request merged**